ide/docker.api/src/org/netbeans/modules/docker/api/DockerAction.java

```diff
@@ -1004,9 +1004,8 @@ public StreamItem fetch() {
         }
     }
 
-    public boolean ping() {
+    public boolean pingWithExceptions() throws IOException, MalformedURLException {
         assert !SwingUtilities.isEventDispatchThread() : "Remote access invoked from EDT";
-        try {
             Endpoint s = createEndpoint();
```

Comment:
The idention is off - please move it to the correct level (one level to the left).